### ISSUE TITLE
Double deployments of blueprint autodeplyments

### ISSUE THREAD

**Issue by jlewi** [kubeflow/testing#666]
The blueprint autodeployments appear to always be deploying two deployments at once e.g we have the following 2 deployments at the same commit spaced 5 seconds apart.

```

cp-blueprint-master | kf-vbp-0513-03c | 1:40:50.981820 | deploy-kf-masterf2fxq | 6f16dce |   | gcloud --project=kubeflow-ci-deployment container clusters get-credentials --region=us-central1 kf-vbp-0513-03c
-- | -- | -- | -- | -- | -- | --
gcp-blueprint-master | kf-vbp-0513-c62 | 1:40:55.981659 | deploy-kf-master5cbqz | 6f16dce |   | gcloud --project=kubeflow-ci-deployment container clusters get-credentials --region=us-central1 kf-vbp-0513-c62
```

**Comment by jlewi** [comment]
It looks like the problem is that when the reconciler starts we end up loading the same YAML file for the pipelinerun twice.

```
Loaded pipelineruns: /app/config/..2020_06_09_19_59_52.425911132/pipeline-run-deploy-blueprint-master.yaml /app/config/pipeline-run-deploy-blueprint-master.yaml
```

It looks like this is because the way the configmap gets mounted there are multiple symlinks.

**Comment by jlewi** [comment]
Fixed by #679. Confirmed by looking at
https://kf-ci-v1.endpoints.kubeflow-ci.cloud.goog/tekton/#/namespaces/auto-deploy/pipelineruns

That pipelineruns are 5 minutes apart so no double deployment.
